### DISCUSSION TITLE
SEO?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-uk-old#117] [Ideas]
Anything we can do to boost SEO? Perhaps unsurprisingly given our low traffic, but we're not showing up anywhere when searching for [policyengine] [policy engine] [ubi center policyengine] [policyengine uk] etc.

**Comment by nikhilwoodruff** [comment]
Seems relevant: https://developers.google.com/search/docs/beginner/seo-starter-guide
